src/interfaces.ts

```diff
@@ -330,18 +330,18 @@ export interface IConversationDetails {
     description?: string;
     roles: IConversationRoles;
     isPublic: boolean;
-    participants?: IConversationParticipant[];
 }
 
 /**
  * Conversation details interface
  */
 export interface IConversationDetails2 extends IConversationDetails {
```

Comment:
It was there because those parameters were returned from the service after the conversation was created...

User populates an IConversationDetails to create and IConversationDetails2  is returned from this call and also the get conversations response ...

Yes I guess we could just amalgamate the two tbh ...  (  add the additional parameters in as optional parameters )